Packs/HarfangLabEDR/ReleaseNotes/1_1_0.md

```diff
@@ -0,0 +1,15 @@
+
+#### Integrations
+##### HarfangLab EDR
+- Updated the Docker image to: *demisto/python3:3.10.5.31928*.
+- Added the *filesize* argument to **IOC search job**. When searching a file based on its name or hash, the *filesize* will take into account instead of the *hash_filesize*.
```

Comment:
```suggestion
- Added the *filesize* argument to **IOC search job**. When searching for a file based on its name or hash, *filesize* will be taken into account instead of the *hash_filesize*.
```